plenum/test/consensus/view_change/test_sim_view_change.py

```diff
@@ -57,10 +60,35 @@ def calc_committed(view_changes):
     return committed
 
 
-@pytest.mark.parametrize("seed", range(200))
-def test_view_change_completes_under_normal_conditions(seed):
-    random = DefaultSimRandom(seed)
-    check_view_change_completes_under_normal_conditions(random)
+@pytest.fixture(params=[(0, 0.6), (1, 2)])
+def set_latency(pool_committed, request, tconf):
+    min, max = tuple(int(param * tconf.NEW_VIEW_TIMEOUT) for param in request.param)
```

Comment:
min and max are reserved words (functions), so it's better to not use them as variables.

Reply:
fixed